### PR TITLE
docs: Normalizing `docker-machine config` references to `docker-machine env`

### LINKED ISSUE #762: Use $(docker-machine env) in getting started guide
It's confusing that the getting started guide switches between `$(docker-machine config)` and `$(docker-machine env)`. We should show you just one way to do it in the getting started guide. My vote is for `$(docker-machine env)` because that's how you're more likely going to do it when working normally.

### PR DESCRIPTION
**Pull request by hairyhenderson**
Fixes #762

Signed-off-by: Dave Henderson <Dave.Henderson@ca.ibm.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -375,7 +378,7 @@ Show the Docker client configuration for a machine.
 
 ```
 $ docker-machine config dev
---tls --tlscacert=/Users/ehazlett/.docker/machines/dev/ca.pem --tlscert=/Users/ehazlett/.docker/machines/dev/cert.pem --tlskey=/Users/ehazlett/.docker/machines/dev/key.pem -H tcp://192.168.99.103:2376
+--tls --tlscacert="/Users/ehazlett/.docker/machines/dev/ca.pem" --tlscert="/Users/ehazlett/.docker/machines/dev/cert.pem" --tlskey="/Users/ehazlett/.docker/machines/dev/key.pem" -H tcp://192.168.99.103:2376
```
Comment (ehazlett) [inline docs/index.md]: I think this should also be `--tlsverify` now.

Comment (hairyhenderson) [inline docs/index.md]: Fixed!

**Comment by ehazlett** [comment]
LGTM

**Comment by bfirsh** [comment]
nice!

**Pull request merged**